cms/management/commands/activation_reminder_mail.py

```diff
@@ -0,0 +1,79 @@
+#code to send automated reminder mails to registerd users for activating their account
+#to run this file please follow below instructions:
+#1. Go to project directory
+#2. run "python manage.py activation_reminder_mail.py >>reminder_mail_success_log.txt"
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Third Party Stuff
+from django.core.management.base import BaseCommand
+from django.db import transaction as tx
+from django.contrib.auth.models import User
+from cms.models import Profile
+from config import *
```

Comment:
Instead of `from config import *`

You should be using 
```
from django.conf import settings

# then use it like 
settings.NO_REPLY_EMAIL
```

Also, check what is the value of  `DEFAULT_EMAIL_FROM` [1], you can use the same instead of introducing a new variable. 

[1] https://docs.djangoproject.com/en/1.10/ref/settings/#std:setting-DEFAULT_FROM_EMAIL